components/case/caseController.js

```diff
@@ -118,7 +118,7 @@ const addCaseArchive = async (req, res, next) => {
           )
         ) {
           // case data is same, no need to update
-          res.status(200).end();
+          res.status(200).json(existingCase);
```

Comment:
is there a swagger for nabu that needs updating to reflect this change?

Reply:
The swagger already shows the case info being returned for a 200 status